### PR TITLE
Additional Props added and README modified

### PR DESCRIPTION
**Pull request by jasurkurbanovinit**
**Following Changes Added:**
1) Additional Props ( namely  `borderWidth`. `borderRadius`,`borderColor` )
   This makes easy for everyone to use them, instead of creating custom styles. It is better to give options for public specific one 
  as well. Everyone will choose by themselves. Sometimes, they use this library for making quick demo where, just changing 
 `borderRadius` will be enough.
2) Props name changed: 
    `checkboxSize` => `size` ( `checkboxSize` was written on README, but wasn't even added in the project)
    `textColor`=> `color` (everybody know that `color` prop is TextStyle props, this is widely used practice. Just keeping it simple for everyone. also description added in README )
    `disableTextDecoration` => textDecoration (too long prop name, since textDecoration props is used widely for drawing line it is better to keep it as textDecoration . README it is for help in case of misunderstanding )
3) I noticed inside project some props are not typed. For example: `iconComponent` and `onPress`. So I added type for them.
4) README updated with new props with their descriptions, I rearranged them for better readability as well.
5) Removed `BouncyCheckbox.style.js`. Because this library is not so big to make another file for styling. Even though, it is good from making everything clean. But offering easiness for reading code, overweights code clearness. Because, in order to see the style I have to open new tab, and find which props where goes. It really tedious for small libraries. Making style in place help better for finding props and its values. Even popular libraries such as `react-native-progress` uses this practice. 
6) It is written in README that project styled with Prettier. But within a project I could find any `.prettierrc.js` file. But in my local environment I use Prettier that's why my Prettier configuration deletes all semicolumns. I guess  even semi columns is good practice, eliminating it better. Since in my opinion we should strive to go forward not backward. 

Thank you. Looking forward hearing you.

### CODE REVIEW

File: lib/src/BouncyCheckbox.js [hunk 1]
```diff
@@ -1,130 +1,153 @@
...
+                justifyContent: 'center',
+                borderColor,
+                backgroundColor: checked ? fillColor : unfillColor
+              },
+              iconStyle
+            ]}
+        >
+          {iconComponent || <Icon size={iconSize} name={iconName} type={iconType} color={iconColor} />}
+        </Animated.View>
+    )
```
Comment (WrathChaos) [inline lib/src/BouncyCheckbox.js:72]: 5. from your comment. 
Actually I don't believe that inline styling is a good practice. It is way too much code to read and decrease the code readability. That's why separated styling is a better choice for me, it is even more controllable.

**Comment by jasurkurbanovinit** [comment]
@WrathChaos I understood from your comments followings:

1) Styles should be separated as it was.
2) Props names that I changed should be reverted back as it was

Am I right ?

File: lib/src/BouncyCheckbox.js (hunk 1 above)
Comment (jasurkurbanovinit) [reply lib/src/BouncyCheckbox.js:72]: Got you will not use inline styling for your projects

**Comment by WrathChaos** [comment]
> @WrathChaos I understood from your comments followings:
> 
> 1. Styles should be separated as it was.
> 2. Props names that I changed should be reverted back as it was
> 
> Am I right ?

1. Correct.
2. Nope, I like the prop names :)

**Comment by jasurkurbanovinit** [comment]
@WrathChaos I commited, please have a look

**Comment by jasurkurbanovinit** [comment]
@WrathChaos Would be great if you upload prettier format that you use in order to format code the same

**Comment by WrathChaos** [comment]
Thank you so much for the contribution @jasurkurbanovinit, I will test it and release a new version asap :)

**Comment by jasurkurbanovinit** [comment]
@WrathChaos You're welcome

**Comment by jasurkurbanovinit** [comment]
@WrathChaos  Can we talk through linkedin or other social platforms ? I have couple questions to you

**Comment by WrathChaos** [comment]
Sure @jasurkurbanovinit, You can reach me on [LinkedIn](https://www.linkedin.com/in/kuray-ogun/)

**Pull request merged**